packages/pyodide-kernel/src/comlink.worker.ts

```diff
@@ -29,6 +29,14 @@ class PyodideDriveFS extends DriveFS {
 }
 
 export class PyodideComlinkKernel extends PyodideRemoteKernel {
+  constructor() {
+    super();
+    this._sendWorkerMessage = (msg: any) => {
+      // use postMessage, but in a format, that comlink would not process.
+      postMessage({ jMsg: msg });
```

Comment:
Wondering if something a bit more explicit like `_kernelMessage` could also work? With or without the leading `_`, but having it could help communicate it's used for "internal" message handling?

Reply:
Ok, but then also the coincident worker needs to rename the change.

Reply:
I meant for the `jMsg` key, we could still keep `_sendWorkerMessage` as the method name:

```suggestion
      postMessage({ _kernelMessage: msg });
```

Reply:
Ok, got it. and it is also unique enough that comlink ignores it. (comlink ignores it if no id field is set...)